src/content/learn/managing-state.md

```diff
@@ -1,30 +1,30 @@
 ---
-title: Managing State
+title: Mengelola State
 ---
 
 <Intro>
 
-As your application grows, it helps to be more intentional about how your state is organized and how the data flows between your components. Redundant or duplicate state is a common source of bugs. In this chapter, you'll learn how to structure your state well, how to keep your state update logic maintainable, and how to share state between distant components.
+Ketika aplikasi Anda berkembang, penting untuk lebih memperhatikan bagaimana *state* Anda diorganisasi dan bagaimana aliran data antara komponen-komponen Anda. *State* yang redundan atau duplikat adalah sumber umum dari bug. Di bab ini, Anda akan belajar bagaimana mengatur *state* dengan baik, bagaimana menjaga logika pembaruan *state* agar mudah dikelola, dan bagaimana berbagi *state* antara komponen-komponen yang jauh.
```

Comment:
`As your application grows` have been discussed in https://github.com/reactjs/id.react.dev/pull/351#discussion_r1179213235 , should we make it same as [`useContext` translation](https://github.com/reactjs/id.react.dev/blob/main/src/content/reference/react/useContext.md?plain=1#L568) or another? @resir014 @mhaidarhanif @zainfathoni .

Reply:
Yes, we need to do it consistently.

The decision for https://github.com/reactjs/id.react.dev/pull/351#discussion_r1179213235 was `Seiring dengan bertumbuhnya aplikasi Anda`, right?

Reply:
```suggestion
Seiring dengan bertumbuhnya aplikasi Anda, penting untuk lebih memperhatikan bagaimana *state* Anda diorganisasi dan bagaimana data mengalir di antara komponen-komponen Anda. *State* yang redundan atau duplikat adalah sumber umum dari *bug*. Di bab ini, Anda akan belajar bagaimana mengatur *state* dengan baik, bagaimana menjaga logika pembaruan *state* agar mudah dikelola, dan bagaimana berbagi *state* antara komponen-komponen yang jauh.
```

I just suggested it here, https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5772140